core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -264,6 +264,15 @@
    void removeCache(String cacheName);
 
    /**
+    * Removes the cache with the given name from the local node.
+    * <p/>
+    * This operation removes the cache and all its contents from memory and cache stores.
+    *
+    * @param cacheName Name of the cache to remove.
+    */
+   void removeCacheLocally(String cacheName);
```

Comment:
Why expose this method publicly? I accept you might want to consolidate logic for locally removing a cache, but I don't see why we should expose this to the user. The user should just call `removeCache` and let the internals decide how it does it.